app/src/main/java/com/cornellappdev/resell/android/ui/components/global/ResellLoadingCard.kt

```diff
@@ -0,0 +1,93 @@
+package com.cornellappdev.resell.android.ui.components.global
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.times
+import com.cornellappdev.resell.android.ui.theme.Stroke
+import com.cornellappdev.resell.android.ui.theme.Wash
+
+@Composable
+fun ResellLoadingCard(
+    modifier: Modifier = Modifier,
+    small: Boolean
+) {
+    val screenWidth = LocalConfiguration.current.screenWidthDp.dp
+    val width = 0.5f * screenWidth
+    val scale = 1.0 / 173.0 * width
+    val height = if (small) 144.73 * scale else 246.4 * scale
+    val contentHeight = if(small) 111.73*scale else 213.4 * scale
```

Comment:
Hmm i'm not too sure what you're going for with the `scale` stuff here. Let's discuss this during work sesh;

The main idea with developing (smartly) with UI is that you want to develop UI that will work with really any screen width, hardcoding widths and heights as infrequently as possible, and using relative stuff like padding and weights to determine component sizes.

Hmm, that might be what you are trying to do here, but these numbers are funky and super specific. Let's definitely discuss this...